setup.py

```diff
@@ -3,15 +3,21 @@
 from setuptools import setup, find_packages
 
 here = os.path.abspath(os.path.dirname(__file__))
-README = open(os.path.join(here, 'README.rst')).read()
-CHANGES = open(os.path.join(here, 'CHANGES.txt')).read()
+try:
+    with open(os.path.join(here, 'README.rst')) as f:
+        README = f.read()
+    with open(os.path.join(here, 'CHANGES.txt')) as f:
+        CHANGES = f.read()
+except IOError:
+    README = CHANGES = ''
 
 requires = [
     'pyramid>=1.3a9',
```

Comment:
Please update pin to not rely on alpha releases.

Reply:
updated